### PR TITLE
docs(grammar): fix grammatical error in footer

### LINKED ISSUE #212: Grammatical error in footer
https://github.com/dyte-in/docs/blob/57fa81562095166c90d86a11099c4190e0c49e33/src/components/homepage/HomeFooter.jsx#L90

This should instead be "Your safety is of utmost importance to us". Here's a screenshot from Grammarly
image

### PR DESCRIPTION
**Pull request by roerohan**
## Description

Changed "Your Safety is utmost important to us" to "Your safety is of utmost importance to us"
Here's what it looks like now:

image

## Resolved issues

Closes #212 

### Before submitting the PR, please take the following into consideration
- [x] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. If you don't have an issue, please create one.
- [x] Prefix your PR title with `feat: `, `fix: `, `chore: `, `docs:`, or `refactor:`.
- [x] The description should clearly illustrate what problems it solves.
- [x] Ensure that the commit messages follow our guidelines.
- [x] Resolve merge conflicts (if any).
- [x] Make sure that the current branch is upto date with the `main` branch.

### CODE REVIEW

**Review by akigugale — COMMENTED** [review]
@roerohan let's make it "Your Security, Our Priority." as discussed with Nireka on slack. Will get this changed across our landing page

**Comment by roerohan** [comment]
@akigugale I've made the requested change. Please take a look.

File: src/components/homepage/HomeFooter.jsx [hunk 1]
```diff
@@ -87,7 +87,7 @@ function Safety({ className }) {
       )}
     >
       <div className="flex flex-1 place-items-center justify-center rounded-2xl bg-white px-4 py-6 font-jakarta font-bold text-gray-500">
-        Your Safety is utmost important to us
+        Your Security, Our Priority.
```
Comment (akigugale) [inline src/components/homepage/HomeFooter.jsx:90]: @roerohan can you add a `<br>` between `Your Security,` & `Our Priority`

Comment (akigugale) [inline src/components/homepage/HomeFooter.jsx:90]: @roerohan can you add a `<br>` between `Your Security,` & `Our Priority`

**Review by akigugale — CHANGES_REQUESTED** [review]

File: src/components/homepage/HomeFooter.jsx (hunk 1 above)
Comment (roerohan) [reply src/components/homepage/HomeFooter.jsx]: Done

**Comment by roerohan** [comment]
I've also updated the github username for @tarushnagpal from `CallMeTarush` to @tarushnagpal

**Review by vaibhavshn — APPROVED** [review]

**Pull request merged**